src/main/resources/archetype-resources/README.md

```diff
@@ -1,83 +1,52 @@
 #set($hash = '#')
-${hash} Initial AEM SPA project template
+${hash} AEM SPA Project
 
-This is a content package for an AEM project set up with the SPA Maven Archetype for AEM (aem-spa-project-archetype).
+**This is a single-page application (SPA) project generated from the [AEM SPA Project Archetype](https://github.com/adobe/aem-spa-project-archetype).**
 
-${hash}${hash} Modules of this project:
+${hash}${hash} Modules
 
-* [core](core/): OSGi bundle containing:
-  * Java classes (e.g. Sling Models, Servlets, business logic)
-* [ui.apps/src/main/content/jcr_root/apps](ui.apps/src/main/content/jcr_root/apps/):
-  * AEM components with their scripts and dialog definitions
-* [ui.content/src/main/content/jcr_root/conf](ui.content/src/main/content/jcr_root/conf/):
-  * AEM content package with editable templates stored at `/conf`
-* [ui.content/src/main/content/jcr_root/content](content/jcr_root/content/):
-  * AEM content package containing sample content (for development and test purposes)
-* [angular-app](angular-app/): Angular application in case frontend chosen is set to be "angular" at project generation
-* [react-app](react-app/): React application in case frontend chosen is set to be "react" at project generation
-* [all](all/): All-in-One package
+- **[`core`](./core/):** OSGi bundle containing Java classes (e.g. Sling models, servlets, business logic)
+- **[`ui.apps`](./ui.apps):** AEM components with their scripts and dialog definitions
+- **[`ui.content`](./ui.apps/):**
+  - [`ui.content/src/main/content/jcr_root/conf`](./ui.apps/src/main/content/jcr_root/conf/): AEM content package with editable templates stored at `/conf`
```

Comment:
It seems to try to open this link -
https://github.com/samuelmeuli/aem-spa-project-archetype/blob/readmes/src/main/resources/archetype-resources/ui.apps/src/main/content/jcr_root/conf

which doesn't exist.

Same is the case for below link for `/content`